cogs/fun.py

```diff
@@ -312,5 +312,13 @@ async def jokes(self, ctx):
         ]
         await ctx.send(random.choice(allJokes))
 
+    ## @story{58} The bot returns a random xkcd comic between 1 and 2400
```

Comment:
Good commenting. I am curious about the wording of "returns a random xkcd comic between 1 and 2400." Is that the number of comics available from the source site you're using?

Reply:
xkcd posts new comics three times a week on Monday, Wednesday, and Friday. Yesterday's comic is number 2455. So the limit would have to be periodically increased, though not frequently (100 new comics per 33 1/3 weeks).